backend/api/views/box.py

```diff
@@ -5,18 +5,33 @@
 from StringIO import StringIO
 from boxsdk.exception import BoxAPIException
 
+from boxsdk import JWTAuth
+from boxsdk import Client
 
-def auth_user():
+"""
+TODO:   AUTHENTICATE USERS USING JWT
+        RETRIEVE INFORMATION FROM BACKEND TABLE
+        PUSH INFORMATION TO FRONTEND
+"""
+
+
+def create_client():
     """
-    Authenticate the user using the client id, secret, and access token.
+    Authenticate the user using the JWT.
     ### Return box client
     """
-
-    oauth2 = OAuth2(CLIENT_ID, CLIENT_SECRET, access_token=ACCESS_TOKEN)
-    client = Client(oauth2)
+    sdk = JWTAuth.from_settings_file("/171399529_73anvn29_config.json")
+    client = Client(sdk)
     return client
 
 
+SPACE = 1073741824
```

Comment:
Is there something to avoid hard coding these credentials into authentication? The reason is that I want to have users easily be able to change and modify the file to read credentials from. Or if possible is there some way we can fetch the changing API key and automatically include it? If not, then for now we can ignore this.